vgstation13.dme

```diff
@@ -2123,7 +2124,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_box.dm"
```

Comment:
Travis needs to fail PRs that do this.
We've already had one get merged without anyone catching it.